front/app/modules/index.ts

```diff
@@ -73,7 +73,7 @@ export default loadModules([
   },
   {
     configuration: nativeSurveyConfiguration,
-    isEnabled: CL_CONFIG['modules']['free/native_surveys'],
+    isEnabled: true,
```

Comment:
We might not need this change since we have already set it [here](https://github.com/CitizenLabDotCo/citizenlab/blob/cl-1253-refactor-ideas-add-survey-handling/citizenlab.config.json#L18).

It should be available to rerunning docker